run.sh

```diff
@@ -1,9 +1,11 @@
+docker run --rm -d -p 6379:6379 --name redis  -v $PWD/data/redis:/data redis
+
+sleep 2
+
 . ./metatron.env
 
 export FLASK_APP=vault
 export FLASK_ENV=development
 export SECRET_KEY='scully:sw33tp0tat0'
```

Comment:
Not sure the `SECRET_KEY` is necessary. Can we remove it? If not, we should probably move it into metatron.env

Reply:
It's just the 2 previous sh scripts combined that already existed.

Reply:
True but the question stands. Can we get rid of the key or should we move it?

Reply:
OK I will do it